package.json

```diff
@@ -47,7 +47,6 @@
     "big-number": "^2.0.0",
     "bl": "^5.0.0",
     "bn": "^1.0.5",
-    "colors": "^1.4.0",
```

Comment:
Please add this commit to open issue related to colors and close it

Reply:
Done